cfme/cloud/provider/__init__.py

```diff
@@ -267,3 +267,19 @@ def wait_for_a_provider():
     wait_for(lambda: int(view.entities.paginator.items_amount), fail_condition=0,
              message="Wait for any provider to appear", num_sec=1000,
              fail_func=view.browser.refresh)
+
+
+def wait_for_cloud_relationship_refresh(provider):
```

Comment:
**Required**: This is an identical function for both providers, suggest move it to cfme/common/provider.py